examples/triggers/workflow-template-trigger.yaml

```diff
@@ -0,0 +1,50 @@
+# Example of a WorkflowTemplate trigger in a Sensor, triggered by a GitHub webhook event. The WorkflowTemplate is triggered when a push event is received from a GitHub repository. The WorkflowTemplate is passed parameters from the webhook event payload, including the repository URL, branch, user name, and email address.
+apiVersion: argoproj.io/v1alpha1
+kind: Sensor
+metadata:
+  name: github-push-workflow-sensor # Sensor Name
+  namespace: argo-events # Namespace
+spec:
+  dependencies:
+    - name: github-webhook # Event source dependency for the sensor
+      eventSourceName: github-push-webhook # Event Source Name
+      eventName: github # Name of the webhook event
+  triggers:
+    - template:
+        name: webhook-workflow-trigger # Trigger Name
+        k8s:
```

Comment:
Should this not use `argoWorkflow`? I see a lot of examples of manipulating workflows with `k8s` directly but it seems like the `argoWorkflow` form is specific to this and that's what the project should steer towards?

Reply:
Updated to use `argoWorkflow`